clients/js/package.json

```diff
@@ -44,28 +44,27 @@
   "dependencies": {
     "@ethersproject/abstract-provider": "^5.7.0",
     "@ethersproject/abstract-signer": "^5.7.0",
-    "@ethersproject/address": "^5.7.0",
     "@ethersproject/bignumber": "^5.7.0",
     "@ethersproject/bytes": "^5.7.0",
-    "@ethersproject/hash": "^5.7.0",
     "@ethersproject/providers": "^5.7.1",
     "@ethersproject/rlp": "^5.7.0",
-    "@ethersproject/transactions": "^5.7.0",
     "@oasisprotocol/deoxysii": "^0.0.5",
     "cborg": "^1.9.5",
+    "ethers6": "npm:ethers@^6.6.1",
     "js-sha512": "^0.8.0",
     "tweetnacl": "^1.0.3",
     "type-fest": "^2.19.0"
   },
   "devDependencies": {
+    "@ethersproject/transactions": "^5.7.0",
     "@types/jest": "^28.1.8",
     "@types/node": "^18.7.18",
     "@types/node-fetch": "^2.6.2",
     "@typescript-eslint/eslint-plugin": "^5.38.0",
     "@typescript-eslint/parser": "^5.38.0",
     "eslint": "^8.23.1",
     "eslint-config-prettier": "^8.5.0",
-    "ethers": "^5.7.1",
+    "ethers5": "npm:ethers@^5.7.2",
```

Comment:
Do we really want to keep ethers5? One can simply use the previous version of `@oasisprotocol/sapphire-paratime`, no?

Reply:
> One can simply use the previous version of @oasisprotocol/sapphire-paratime, no?

yes, but I don't want to create a new major version of this library yet. the current interface still hasn't hit its breaking point. I suspect people will not enjoy the slightly larger bundle size this causes, but maybe nobody will notice. Let's hold off on a v2 until the consensus is in on what the replacement should be.